src/coreclr/nativeaot/docs/compiling.md

```diff
@@ -4,18 +4,7 @@ This document explains how to compile and publish your project using Native AOT
 
 ## Add ILCompiler package reference
 
-To use Native AOT with your project, you need to add a reference to the ILCompiler NuGet package containing the Native AOT compiler and runtime. Make sure the `nuget.config` file for your project contains the following package sources under the `<packageSources>` element:
-```xml
-<add key="dotnet7" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7/nuget/v3/index.json" />
-<add key="nuget" value="https://api.nuget.org/v3/index.json" />
-```
-
-If your project has no `nuget.config` file, it may be created by running
-```bash
-> dotnet new nugetconfig
-```
-
-from the project's root directory. New package sources must be added after the `<clear />` element if you decide to keep it.
+To use Native AOT with your project, you need to add a reference to the ILCompiler NuGet package containing the Native AOT compiler and runtime.
```

Comment:
Starting from preview 5, you don't need to do this manually. Just add `<PublishAot>true</PublishAot>` in PropertyGroup, and SDK will pull the required packs (similar to `PublishReadyToRun` or `PublishSingleFile`, we don't add their packages manually).